Source/Core/VideoBackends/D3D/D3DBase.cpp

```diff
@@ -224,6 +224,59 @@ bool SupportsTextureFormat(DXGI_FORMAT format)
   return (support & D3D11_FORMAT_SUPPORT_TEXTURE2D) != 0;
 }
 
+bool SupportsLogicOp(u32 adapter_index)
+{
+  // Use temporary device if we don't have one already.
+  Common::DynamicLibrary temp_lib;
+  ComPtr<ID3D11Device1> temp_device1;
```

Comment:
Can this just be `ComPtr<ID3D11Device1> temp_device1 = device1;`?

Reply:
In theory it can be, but I wanted to make it explicit that `device1` shall not be used at all if `device` is not initialized.